src/Components/project3/AddNewPerson.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from "react";
+import { Container, Form, Button, Row, Col } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import AlertMessage from "./Alert";
+import IUser, { initialUserValues } from "./Interfaces";
+import ProcessesWebErrorStatus from "./ProcessesWebErrorStatus";
+
+const AddNewPerson: React.FC = () => {
+    const [user, setUser] = useState<IUser>(initialUserValues);
+    const [showAlert, setShowAlert] = useState(false);
+    const [alertMessage, setAlertMessage] = useState("");
```

Comment:
instead of using alertMessage and alertVariant, you can keep in state an Alert object, with 2 props: message and variant